soroban-sdk/src/deploy.rs

```diff
@@ -22,11 +18,10 @@
 //! #
 //! # #[contractimpl]
 //! # impl Contract {
-//! #     pub fn f(env: Env) {
-//! # let wasm = [0u8; 32];
-//! # let salt = [0u8; 32];
-//! let deployer = env.deployer().with_current_contract(&salt);
-//! let contract_id = deployer.deploy(&wasm);
+//! #     pub fn f(env: Env, wasm_id: BytesN<32>) {
+//! #         let salt = [0u8; 32];
+//! #         let deployer = env.deployer().with_current_contract(&salt);
+//! #         let contract_id = deployer.deploy(&wasm_id);
```

Comment:
These lines were uncommented so that they are visible in the Rust docs. Is there a reason they are being hidden? They're the lines we're demonstating to the user.
```suggestion
//! let deployer = env.deployer().with_current_contract(&salt);
//! let contract_id = deployer.deploy(&wasm_id);
```

Reply:
Oh, I just thought that was unintentionally incorrect formatting, I didn't know that we show only a snippet of the example...

Reply:
It's a pretty cool feature of rust docs. It helps to shave our examples down while keeping them fully functional.